lib/core/test/bench/db/Main.hs

```diff
@@ -536,7 +558,7 @@ mkTxHistory numTx numInputs numOutputs range =
         )
     | !i <- [1..numTx]
     , let inps = (mkInputs i numInputs)
-    , let outs = (mkOutputs i numOutputs)
+    , let outs = (mkOutputs' i numOutputs)
```

Comment:
**Request**: Can we rename the instances of `mkOutputs'` to `mkOutputs`?

It would reduce the size of the diff.

I did a search and couldn't find any instances of `mkOutputs`, so it should be safe to rename.